libs/ui-components/src/components/Device/DevicesPage/EnrollmentRequestList.tsx

```diff
@@ -77,12 +77,8 @@ const EnrollmentRequestList = ({ refetchDevices, isStandalone }: EnrollmentReque
     },
   });
 
-  if (isLoading) {
-    return <Spinner size="md" />;
-  }
-
-  if (pendingEnrollments.length === 0) {
-    return isStandalone ? <ErEmptyState /> : null;
+  if (!isStandalone && (isLoading || pendingEnrollments.length === 0)) {
```

Comment:
I have made some adaptations in this area after rebasing the PR for device pagination.

I wanted to keep the view of the current table whenever some content had been loaded, and we're fetching a new page. The pagination content will have a small spinner, but the table would be keep showing at the current page.

WDYT?
 https://github.com/flightctl/flightctl-ui/pull/178/files#diff-978cbe9c1ef3453ed71f0d89f3551bb56a7654c1995907f14ff77b64454de47fR77